tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -82,6 +83,37 @@ def test_arxiv_fulltext_download_logs_on_success():
 
 
 @pytest.mark.httpretty
+def test_arxiv_fulltext_download_not_ready():
```

Comment:
Let's make it a real sentence: `test_arxiv_fulltext_download_retries_when_fulltext_is_not_ready`